src/serverdlg.cpp

```diff
@@ -106,16 +106,16 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
     edtLocationCity->setAccessibleName ( tr ( "City where the server is located line edit" ) );
 
     // location country
-    QString strLocCountry = "<b>" + tr ( "Location country" ) + ":</b> " +
-                            tr ( "The country in "
-                                 "which this server is located can be set here. If a country is "
+    QString strLocCountry = "<b>" + tr ( "Country/Region" ) + ":</b> " +
+                            tr ( "The country or region in "
+                                 "which this server is located can be set here. If a location is "
```

Comment:
As per style guide, we should try to put things into active case for better readability. So this would be:

"Set the country or region in which this server is located here. Clients will show this location in their connect dialog's server list."

Reply:
See [here](https://github.com/jamulussoftware/jamulus/pull/1927/files#r684075970).